api/__init__.py

```diff
@@ -1,7 +1,12 @@
-import connexion
-import requests
+import os
 
-connexion_app = connexion.App(__name__)
+import connexion
 
-from item_tracker_core import handlers
+app = connexion.App(__name__, specification_dir='./')
 
+app.add_api('openapi.yml')
```

Comment:
this should be in `__main__.py` one shouldn't really be trying to run a module like a program in this way